web/server/web_client_cache.c

```diff
@@ -188,15 +188,15 @@ struct web_client *web_client_get_from_cache_or_allocate() {
 #ifdef ENABLE_HTTPS
```

Comment:
Changes here don't belong in this PR

Reply:
I compared the master with this, and it looks like the change was only space, because this line is present in the master.

Reply:
I remembered the changes that I made, I executed a new pull for it!